src/role/index.js

```diff
@@ -36,12 +36,16 @@ class Role extends EventEmitter {
         await testRun.executeCommand(navigateCommand);
 
         try {
+            testRun.debugging = debugging;
             await this.initFn(testRun);
         }
         catch (err) {
             this.initErr = err;
         }
 
+        testRun.previousDebuggingState = testRun.debugging;
```

Comment:
We need, because during execution role's actions step-by-spep user can click on button `Resume test` so `testRun.debugging` flag should be reasigned